src/lib/core/say.c

```diff
@@ -1001,12 +1001,9 @@ format_syslog_header(char *buf, int len, int level,
 	int total = 0;
 
 	/* Format syslog header according to RFC */
-	int prio = level_to_syslog_priority(level);
-#if defined(__OpenBSD__)
-#define LOG_MAKEPRI(fac, pri)        (((fac) << 3) | (pri))
-#endif
-	SNPRINT(total, snprintf, buf, len, "<%d>",
-		LOG_MAKEPRI(8 * facility, prio));
+	int prio = (((facility) << 3) | level_to_syslog_priority(level));
```

Comment:
Nit: The parentheses around `facility` and around the whole rhs (right hand statement) are redundant.

Reply:
Fixed.